pages/management/ManagementHandler.php

```diff
@@ -258,17 +258,13 @@ public function workflow($args, $request)
         $disableSubmissionsForm = new \PKP\components\forms\context\PKPDisableSubmissionsForm($contextApiUrl, $locales, $context);
         $emailSetupForm = $this->getEmailSetupForm($contextApiUrl, $locales, $context);
         $metadataSettingsForm = new \APP\components\forms\context\MetadataSettingsForm($contextApiUrl, $context);
-        $reviewGuidanceForm = new \APP\components\forms\context\ReviewGuidanceForm($contextApiUrl, $locales, $context);
-        $reviewSetupForm = new PKPReviewSetupForm($contextApiUrl, $locales, $context);
```

Comment:
I think there will be a `use` statement for `PKPReviewSetupForm` that needs to be removed from the top of this file.